olaf/scripts/runner.py

```diff
@@ -0,0 +1,36 @@
+import os
+import spacy
+from abc import ABC, abstractmethod
+from olaf import Pipeline
+from olaf.repository.corpus_loader import TextCorpusLoader
+
+
+class Runner(ABC):
+    def __init__(self, model_name="en_core_web_md"):
```

Comment:
specify the type of `model_name`, not a default value and add `corpus_path` with its type as parameter